source/scss/_pf-shame.scss

```diff
@@ -17,3 +17,14 @@ limitations under the License.
 // Patternfly Shame
 // --------------------------------------------------
 // The nest fot all our shamefull code.
+
+
+// This is a bug on bootstrap alpha 6, it is fixed on master. Remote thjs lines
```

Comment:
Typo, but given this is the Shame file, I'm totally fine with leaving it. ;-)
Also "for" and "shameful" on the previous line.